app/views/admin/loans/_timeline.html.slim

```diff
@@ -0,0 +1,11 @@
+#timeline-table
+  = render "admin/timeline/main", project: @loan
+
+javascript:
+  $(function() {
+    new MS.Views.TimelineTableView({
+      projectId: #{json @loan.id},
+      stepModal: new MS.Views.ProjectStepModalView()
+    });
+  });
+
```

Comment:
This file is now basically identical to `basic_projects/timeline`.

I get why we still need this file due to your nice loop thing but perhaps at least you could move the JS initialize into `timeline/main`? I don't see a need to duplicate it.